acousticbrainz/data/user.py

```diff
@@ -0,0 +1,81 @@
+import psycopg2
+from flask import current_app
+from flask_login import UserMixin
+from werkzeug.exceptions import BadRequest, ServiceUnavailable
+
+
+def create(musicbrainz_id):
+    try:
+        connection = psycopg2.connect(current_app.config['PG_CONNECT'])
+        cursor = connection.cursor()
+        cursor.execute('INSERT INTO "user" (musicbrainz_id) VALUES (%s) RETURNING id',
+                       (str(musicbrainz_id),))
+        connection.commit()
+        new_id = cursor.fetchone()[0]
+    except psycopg2.ProgrammingError, e:
+        raise BadRequest(str(e))
+    except psycopg2.IntegrityError, e:
+        raise BadRequest(str(e))
+    except psycopg2.OperationalError, e:
+        raise ServiceUnavailable(str(e))
+
+    return new_id
+
+
+def get(id):
+    try:
+        connection = psycopg2.connect(current_app.config['PG_CONNECT'])
+        cursor = connection.cursor()
+        cursor.execute('SELECT id, created, musicbrainz_id FROM "user" WHERE id = %s',
+                       (str(id),))
+    except psycopg2.IntegrityError, e:
+        raise BadRequest(str(e))
+    except psycopg2.OperationalError, e:
+        raise ServiceUnavailable(str(e))
+
+    if cursor.rowcount > 0:
```

Comment:
why might there be two or more rows as a result of this query? Should it silently ignore this fact, or cause an error?